src/main/java/emissary/pool/PayloadLauncher.java

```diff
@@ -21,6 +21,7 @@ public class PayloadLauncher {
     private static final Logger logger = LoggerFactory.getLogger(PayloadLauncher.class);
 
 
+    @SuppressWarnings("CatchingUnchecked")
```

Comment:
why does this method catch Exception and not RuntimeException?

Reply:
`pool.borrowAgent()` throws Exception